listenbrainz/webserver/static/js/src/MissingMBData.tsx

```diff
@@ -0,0 +1,338 @@
+                  const listen = {
+                    listened_at:
+                      new Date(`${data.listened_at}Z`).getTime() / 1000,
+                    user_name: user.name,
+                    track_metadata: {
+                      artist_name: data.artist_name,
+                      track_name: data.recording_name,
+                      release_name: data?.release_name,
+                    },
+                  };
+                  const additionalActions = (
+                    <>
+                      <ListenControl
+                        icon={faPlus}
+                        title=""
+                        // eslint-disable-next-line react/jsx-no-bind
+                        action={this.submitMissingData.bind(this, listen)}
+                      />
+                    </>
+                  );
+                  return (
+                    <ListenCard
+                      key={`${data.recording_name}-${data.artist_name}-${data.listened_at}`}
+                      showTimestamp
+                      showUsername={false}
+                      newAlert={newAlert}
+                      listen={listen}
+                      additionalActions={additionalActions}
+                    />
+                  );
+                })}
+              </div>
+              <ul className="pager" style={{ display: "flex" }}>
+                <li
+                  className={`previous ${
+                    currPage && currPage <= 1 ? "hidden" : ""
+                  }`}
+                >
+                  <a
+                    role="button"
+                    onClick={this.handleClickPrevious}
+                    onKeyDown={(e) => {
+                      if (e.key === "Enter") this.handleClickPrevious();
+                    }}
+                    tabIndex={0}
+                  >
+                    &larr; Previous
+                  </a>
+                </li>
+                <li
+                  className={`next ${
+                    currPage && currPage >= totalPages ? "hidden" : ""
+                  }`}
+                  style={{ marginLeft: "auto" }}
+                >
+                  <a
+                    role="button"
+                    onClick={this.handleClickNext}
+                    onKeyDown={(e) => {
+                      if (e.key === "Enter") this.handleClickNext();
+                    }}
+                    tabIndex={0}
+                  >
+                    Next &rarr;
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div
+            className="col-xs-12"
+            /* eslint-disable no-dupe-keys */
+            style={{
+              position: "-webkit-sticky",
+              // @ts-ignore
+              position: "sticky",
+              top: 20,
+            }}
+            /* eslint-enable no-dupe-keys */
+          >
+            <BrainzPlayer
+              direction={direction}
```

Comment:
Will it be okay if this line is removed during the tackling of merge conflict? Since changing this rn here at my branch will cause trouble to test the page locally.